db/models/user.js

```diff
@@ -9,6 +9,8 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       this.belongsToMany(models.category, { through: "user_categories" });
+      this.hasMany(models.request);
+      this.hasMany(models.donation);
```

Comment:
Add this.belongsToMany  to book as donor and bene through donation
add this.belongsToMany  to donation throught request